discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+# System Settings [](id=system-settings)
+
+@product@ is modular, meaning it's composed of many applications, which are
+divided into even smaller "chunks" of functionality. The applications, and
+sometimes even code chunks, are configurable at several scopes, as discussed in
+the introductory article for this section. 
+
+To make configuration changes to an application in a system-wide fashion, use
+the System Settings application at Control Panel &rarr; Configuration &rarr;
+System Settings.
+
+![Figure 1: System Settings are accessed through the Control Panel.](../../../images/system-settings-categories.png)
+
+## Editing the Default Configuration [](id=editing-the-default-configuration)
+
+In System Settings, the configuration options are organized into logical
+sections (for example, Content) and categories (for example, Workflow) based on
+their functionality. There's also a Search bar to make finding configuration
+entries easier. 
```

Comment:
Add "You can search for the name of a specific configuration entry or even an specific field by its name"